stdlib/pathname.rb

```diff
@@ -87,23 +87,23 @@ def plus(path1, path2) # -> path # :nodoc:
     prefix2 = path2
     index_list2 = []
     basename_list2 = []
-    while r2 = chop_basename(prefix2)
+    while (r2 = chop_basename(prefix2))
       prefix2, basename2 = r2
       index_list2.unshift prefix2.length
       basename_list2.unshift basename2
     end
     return path2 if prefix2 != ''
     prefix1 = path1
-    while true
+    loop do
```

Comment:
I'll guess 🤖 👮 has a good reason for doing this but in opal while true looks faster:

```
Warming up --------------------------------------
          while true   407.000  i/100ms
                loop   184.000  i/100ms
Calculating -------------------------------------
          while true      7.088k (± 2.6%) i/s -     35.409k in   4.999414s
                loop      1.970k (± 2.6%) i/s -      9.936k in   5.046192s

Comparison:
          while true:     7087.7 i/s
                loop:     1970.5 i/s - 3.60x  slower
```

code:

```rb
Benchmark.ips do |x|
  x.report('while true') do
    n = 0
    while true
      n += 1
      break if n == 10000
    end
  end

  x.report('loop') do
    n = 0
    loop do
      n += 1
      break if n == 10000
    end
  end

  x.compare!
end
```